README.md

```diff
@@ -1,8 +1,66 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Sahar Soheilian Esfahani . Jarvis Consulting
 
+Completed an MSc in computer engineering at Shahid Beheshti University in Iran. Performance-driven and result-oriented developer with more than 5 years of experience in database design and project management in automation projects (BPMS). Experienced in leading engineering teams to achieve concrete goals on a strict deadline. Worked as a backend developer and implemeneted RESTful APIs in Django to support the platform's frontend and pre-process data. Skilled in Java, SQL, Python, Django, and docker. Seeking to leverage my expertise and knowledge to work on challenging problems and enhance product capabilities.
```

Comment:
please run a grammarly check.

Please use full complete sentences for your introduction. e.g. 
I have recently completed an ...
I am a performance-driven ...

please also talk about your jarvis experience